content/birds/olu.md

```diff
@@ -0,0 +1,14 @@
+title: Olu Niyi-Awosusi
+sub: Javascript Developer
+bird: olu
+date: 2021-06-07
+image:
+  src: [`birds/bird.jpg`]
```

Comment:
For the page headers, we've been adding actual photos. You can add that in `src/images/birds/` and preview the page at `/authors/olu/` to make sure you like how it lays out. If needed, you can add a `position` attribute to the `image` data, which sets [object-position](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) on the image.